site/index.html

```diff
@@ -1,11 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="icon" type="image/png" href="/C:\Users\felip\OneDrive\Documentos\Sault College\lab-2-25s-luisfe1505/favicon-96x96.png" sizes="96x96" />
+    <link rel="icon" type="image/svg+xml" href="/C:\Users\felip\OneDrive\Documentos\Sault College\lab-2-25s-luisfe1505/favicon.svg" />
+    <link rel="shortcut icon" href="/C:\Users\felip\OneDrive\Documentos\Sault College\lab-2-25s-luisfe1505/favicon.ico" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/C:\Users\felip\OneDrive\Documentos\Sault College\lab-2-25s-luisfe1505/apple-touch-icon.png" />
+    <link rel="manifest" href="/C:\Users\felip\OneDrive\Documentos\Sault College\lab-2-25s-luisfe1505/site.webmanifest" />
```

Comment:
Prefer relative paths instead
```suggestion
    <link rel="icon" type="image/png" href=favicon-96x96.png" sizes="96x96" />
    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png" />
    <link rel="manifest" href="site.webmanifest" />
```